test/libsolidity/semanticTests/constructor/constructor_arguments_external.sol

```diff
@@ -19,7 +19,7 @@ contract Main {
 // compileViaYul: also
 // ----
 // constructor(): "abc", true
-// gas irOptimized: 112563
+// gas irOptimized: 107175
```

Comment:
This is interesting. Why does this change affect the constructor cost at all? And by that much?